<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselCrewEditForm</class>
 <widget class="QDialog" name="VesselCrewEditForm">
  <property name="windowTitle">
   <string>Сведения о судне и экипаже</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelVesselName">
       <property name="text"><string>Название судна:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="vesselName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelIMO">
       <property name="text"><string>IMO / MMSI:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="imoMmsi"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFlag">
       <property name="text"><string>Флаг:</string></property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="flag"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelCrewCount">
       <property name="text"><string>Экипаж (чел):</string></property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="crewCount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text"><string>Сохранить</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text"><string>Отмена</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>